lib/sensu-mutator.rb

```diff
@@ -67,6 +71,15 @@ def self.disable_autorun
       return unless @@autorun
       mutator = @@autorun.new
       mutator.read_event(STDIN)
+
+      TRUTHY_VALUES = %w[1 t true yes y].freeze
+      automap = ENV['SENSU_MAP_V2_EVENT_INTO_V1'].to_s.downcase
+
+      if mutator.config[:map_v2_event_into_v1,] || TRUTHY_VALUES.include?(automap)
```

Comment:
`,` looks like a mistake

Reply:
interesting that this is allowed syntax by ruby. Fixed in newer commits.